src/components/MessagesList/MessagesGroup/Message/Message.vue

```diff
@@ -546,6 +548,12 @@ export default {
 	},
 
 	methods: {
+		lastReadMessageVisibilityChanged(isVisible) {
+			if (isVisible) {
+				this.seen = true
```

Comment:
optional: move this to store or a global space ? (would save having an extra bool for every message)